controllers/openflow/flows.go

```diff
@@ -109,8 +109,14 @@ func (c *controller) syncFlows() error {
 }
 
 func (c *controller) defaultFlows() error {
-	// table entry, for now, always go directly to network policies
+	// arp for the gateway IP should go back to node-local
 	flow := flows.NewFlow().WithTable(tableEntry).
+		WithPriority(41000).WithProtocol("arp").
+		WithNormal()
```

Comment:
Wondering why `NORMAL` and not `LOCAL` here?

Reply:
the idea is to reduce the number of ARP flows, so we rely on OVS learning for ARP traffic on this host, and only forward ARP requests to remote hosts for remote IPs.

Reply:
Ah, I see. But this would result in broadcast right? I also think since this is higher priority it will broadcast all addresses and not just the local ones?

Reply:
not sure about the broadcast, wouldn't it just do normal l2 learning? and you are right about the priority, I missed that.